cmd/cluster-init/generate_job.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	prowapi "k8s.io/test-infra/prow/apis/prowjobs/v1"
+	prowconfig "k8s.io/test-infra/prow/config"
+	utilpointer "k8s.io/utils/pointer"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/jobconfig"
+)
+
+const (
+	LatestImage = api.ServiceDomainAPPCIRegistry + "/ci/applyconfig:latest"
+	LabelRole   = "ci.openshift.io/role"
+	Infra       = "infra"
```

Comment:
```suggestion
	JobRoleInfra       = "infra"
```

Perhaps to make it a little more clear *what* this is?